projects/assets-library/src/icons/icon-library.module.ts

```diff
@@ -91,7 +91,12 @@ const iconsRootPath = 'assets/icons';
         { key: IconType.TriangleUp, url: `${iconsRootPath}/triangle-up.svg` },
         { key: IconType.Yaml, url: `${iconsRootPath}/yaml.svg` },
         { key: IconType.VirtualMachine, url: `${iconsRootPath}/virtual-machine.svg` },
-        { key: IconType.Warning, url: `${iconsRootPath}/warning.svg` }
+        { key: IconType.Warning, url: `${iconsRootPath}/warning.svg` },
+        { key: IconType.Cartesian, url: `${iconsRootPath}/loader-cartesian.svg` },
```

Comment:
Please append 'Loader' to the names. 

Also, Could you please rename Logo to PageLoader?

Reply:
@Christian862 this one might be missed

Reply:
Missed by what ?

Reply:
I mean, did you address this change in latest commit?

Reply:
Yes its been changed